.eslintrc.json

```diff
@@ -1,338 +1,360 @@
-        ],
-        "spaced-comment": [
-            2,
-            "always"
-        ],
-        "use-isnan": "error"
+    "import/extensions": [".js", ".mjs", ".jsx", ".ts", ".tsx", ".d.ts"],
+    "import/external-module-folders": ["node_modules", "node_modules/@types"]
+  },
+  "env": {
+    "es6": true,
+    "browser": true,
+    "node": true
+  },
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
     }
+  },
+  "rules": {
+    "import/prefer-default-export": "off",
+    "react/destructuring-assignment": "off",
+    "react/jsx-filename-extension": "off",
+    "react/no-deprecated": "warn",
+    "react/jsx-no-target-blank": "warn",
+    "react/no-find-dom-node": "warn",
+    "@typescript-eslint/explicit-module-boundary-types": "off",
+    "@typescript-eslint/adjacent-overload-signatures": "error",
+    "@typescript-eslint/array-type": [
+      "error",
+      {
+        "default": "array"
+      }
+    ],
+    "@typescript-eslint/ban-types": [
+      "error",
+      {
+        "types": {
+          "Boolean": "Avoid using the `Boolean` type. Did you mean `boolean`?",
+          "Function": "Avoid using the `Function` type. Prefer a specific function type, like `() => void`.",
+          "Number": "Avoid using the `Number` type. Did you mean `number`?",
+          "Object": "Avoid using the `Object` type. Did you mean `object`?",
+          "String": "Avoid using the `String` type. Did you mean `string`?",
+          "Symbol": "Avoid using the `Symbol` type. Did you mean `symbol`?"
+        }
+      }
+    ],
+    "@typescript-eslint/naming-convention": [
+      "error",
+      {
+        "selector": "default",
+        "format": ["camelCase"],
+        "leadingUnderscore": "allow",
+        "trailingUnderscore": "allow"
+      },
+
+      {
+        "selector": "variable",
+        "format": ["camelCase", "UPPER_CASE"],
+        "leadingUnderscore": "allow",
+        "trailingUnderscore": "allow"
+      },
+
+      {
+        "selector": "typeLike",
+        "format": ["PascalCase"]
+      },
+      { "selector": "class", "format": ["PascalCase"] }
+    ],
+    "@typescript-eslint/consistent-type-assertions": [
+      "error",
+      {
+        "assertionStyle": "as",
+        "objectLiteralTypeAssertions": "never"
+      }
+    ],
+    "@typescript-eslint/explicit-function-return-type": "off",
+    "@typescript-eslint/explicit-member-accessibility": [
+      "error",
+      {
+        "accessibility": "explicit",
+        "overrides": {
+          "accessors": "explicit",
+          "constructors": "no-public",
+          "methods": "explicit",
+          "parameterProperties": "off",
+          "properties": "explicit"
+        }
+      }
+    ],
+    "@typescript-eslint/indent": ["off", 4],
+    "@typescript-eslint/member-delimiter-style": [
+      "error",
+      {
+        "multiline": {
+          "delimiter": "semi",
+          "requireLast": true
+        },
+        "singleline": {
+          "delimiter": "semi",
```

Comment:
This will be a breaking change - everywhere where we were previously mandating that type literals should have commas between members, we will now be mandating we have semicolons.

The semicolon style is apparently preferred in typescrpt according to whoever wrote this rule https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/member-delimiter-style.md . Does anyone know why we opted to enforce commas instead? or feel strongly enough for us to change to semicolons?

Reply:
Imo it should be consistent so even if it's a breaking change we should stick with semicolons (mostly because they're more common and therefore they're less likely to confuse new devs) for single lines as we did for multi lines.

Reply:
What are we doing with this one? 

I think it still should be fairly easy to fix as we can either run `eslint -fix` or override in the project and set it to warning so we can gradually fix it manually